vassal-app/src/main/java/VASSAL/launch/BasicModule.java

```diff
@@ -242,20 +242,23 @@ public String encode(Command c) {
     if (c == null) {
       return null;
     }
-    String s = encodeSubCommand(c);
+    StringBuilder sb = new StringBuilder();
+    sb.append(encodeSubCommand(c));
     String s2;
     Command[] sub = c.getSubCommands();
     if (sub.length > 0) {
-      SequenceEncoder se = new SequenceEncoder(s, COMMAND_SEPARATOR);
+      SequenceEncoder se = new SequenceEncoder(COMMAND_SEPARATOR);
```

Comment:
This isn't right. If `s` contains the separator, it won't get escaped properly now.